cmd/run.go

```diff
@@ -58,6 +59,7 @@ func init() {
 	runCmd.Flags().BoolVar(&vmConfig.Quiet, "quiet", false, "Suppress output from bootc disk creation and VM boot console")
 	runCmd.Flags().StringVar(&diskImageConfigInstance.RootSizeMax, "root-size-max", "", "Maximum size of root filesystem in bytes; optionally accepts M, G, T suffixes")
 	runCmd.Flags().StringVar(&diskImageConfigInstance.DiskSize, "disk-size", "", "Allocate a disk image of this size in bytes; optionally accepts M, G, T suffixes")
+	runCmd.Flags().StringArrayVar(&vmConfig.BindMounts, "bind", nil, "Create a virtiofs mount between host and guest, separated by a `:`")
 }
```

Comment:
why not using "-v/--volume"?, even if not a great name is more coherent with podman

Reply:
Also, maybe it's better if we support multiple "-v", instead of a single "-v" with multiple shared dirs

Reply:
Right, sorry when working on this I'd been in a mindset of porting features from `coreos-assembler run`.  Will change.